static/js/navigation.js

```diff
@@ -58,3 +58,54 @@ function initDropdowItems(itemClass) {
 
 initNavDropdowns('.p-navigation__item--dropdown-toggle');
 initDropdowItems('.p-navigation__dropdown-item');
+
+
+// Init GA tracking
+addGANavEvents("#products", "canonical.com-nav-products");
```

Comment:
I would add one for the "Company" nav item here, given it's a new addition

Reply:
Also - the "Products" list item doesn't have the id, so the event can't fire. Events _are_ being fired, however, so we just need to make these two changes.